tests/hash-backend-integration/src/tests/test-server.ts

```diff
@@ -96,3 +111,37 @@ export const resetToSnapshot = async (snapshotPath: string) => {
 
   await restoreSnapshot(snapshotPath);
 };
+
+const graphContext = createTestImpureGraphContext();
```

Comment:
I'm not sure about us making a client and such at the top level here... generally like to avoid any functional code that's ran when you import a module.

Reply:
The functions are now living in `hash-api` and require a context to be passed in.